Documentation/Signoffs/IsaacJennings-Signoff-MasterControl.md

```diff
@@ -0,0 +1,109 @@
+	of the allowed cubic foot (1 ft x 1 ft x 6 in) [1]–[3].
+	* Autonomy will be achieved by the master controller. All navigation decision will be made in the master controller. 
+
+- C9:The robot shall travel inclines and declines up to 25 degrees
+	* The master controller will send the signal to the motor controller, so it must know when and to what degree any inclines and declines may be. 
+
+- C10:The robot shall turn 360 degrees left and right and move
+	forwards and backwards based on sensor inputs.
+	* The master controller will send commands to enable movement.
+
+	
+- C11: The robot shall have a navigation system that controls
+	movement.
+	* The master controller will use the data from the sensor inputs to make decisions on the movement outputs. 
+	
+- C12:The robot shall know its location within a two-inch
+	tolerance.
+	* The sensor information must be accurately decoded in the master controller to ensure the two-inch tolerance is met. 
+	
+- C13:The robot shall possess a maximum speed of 2 feet per
+	second.
+	* The master controller will send speed commands, so it is necessary that the proper speed that is sent should show at the output. 
+	
+	
+
+## Analysis
+
+
+The following is a table of the Jetson Nano's pins and adapters:
+
+|Name|Quantity|
+|-|-|
+|Micros SD slot| 1|
+|GPIO pins| 40|
+|Micro-USB|1|
+|Gigabit ethernet port| 1|
+|USB 3.0 Port| 4|
+|HDMI output port| 1|
+|DC barrel jack| 1|
+|MIPI CSI-2 camera connectors| 1|
+
+
+The Jetson Nano only has a 40-pin expansion header to interface with sensors, peripherals, and 
+motors, which does not leave an abundanc of extra space. As a result, two of the USB 3.0 ports will be utilized to communicate with
+the system microcontrollers. Serial communication will be sent from the sensors to the 
+master controller, and then the master controller will make decisions based off of its task and the data. 
+Most of the pins will be left for future users to implement tasks. 
+	In terms of current, if the DC barrel jack is used, battery and charging 
+Subsystem will need to supply the Jetson Nano with 5V and 4A to properly power the controller. 
+However, if the micro-USB is used, the current only needs to reach 2A to properly power the controller. 
+Therefore, it is recommended that the micro-USB is used to reduce the current draw on the power system. 
+
+Data Synchronization between multiple microcontrollers will be achieved with the libraries provided on
+the Jetson Nano. Commands like stream.synchronize() will achieve the desired synchronization of data streams. 
+
+To have enough storage for any navigation algorithm that is conceivable on the Jetson Nano, a 256GB SD card will be purchased to ensure the 
+proper amount of space is available for all applications. In order to interface with the Jetson Nano, a keyboard and mouse will also be needed.
+
+## ROS Implementation and Testing
+
+The robot operating system (ROS) was used in the 2023 SECON robot as the master level controller. 
```

Comment:
Please provide references here.

Reply:
A reference to the GitHub was added to the top of the references

Reply:
Could you get the reference number in the description? Is it the first one?